usr/bin/resize-root-file-system.sh

```diff
@@ -3,7 +3,14 @@
 set -x
 
 # Example output: "/dev/mmcblk0p5" (SD card or eMMC), "/dev/nvme0n1p5" (NVMe), or "/dev/sda5" (SATA)
-root_partition=$(mount | grep 'on \/ ' | awk '{print $1}')
+root_partition=$(mount | grep 'on / ' | awk '{print $1}')
+overlay_detected="false"
+
+# When using OCI builds (not rpm-ostree builds), the file system is overlayed sooner.
+if [ "${root_partition}" == "overlay" ]; then
+    root_partition=$(mount | grep 'on /var ' | awk '{print $1}')
```

Comment:
Instead of adding all of this extra code, I could have simply done a check at the very beginning for `/var` . I.e., `root_partition=$(mount | grep 'on /var ' | awk '{print $1}')`. However, I wanted to modify the code so that this can be leveraged by others in the community if they want to resize a Btrfs root file systems in their own OS. It's common for advanced users to place `/var` onto a separate mount other than `/`.